lib/linguist/languages.yml

```diff
@@ -2543,6 +2543,15 @@ MiniD:
   tm_scope: none
   ace_mode: text
   language_id: 231
+MiniZinc:
+  type: programming
+  color: "#1295e6"
+  extensions:
+  - ".mzn"
+  - ".dzn"
```

Comment:
[`.dzn` files](https://github.com/search?p=4&q=extension%3Adzn+NOT+dfjdfkjgjf&ref=searchresults&type=Code&utf8=%E2%9C%93) look more like data files. Are they?